storage/src/vespa/storage/persistence/filestorage/filestorhandlerimpl.h

```diff
@@ -194,7 +200,7 @@ class FileStorHandlerImpl : private framework::MetricUpdateHook,
             // Disperse bucket bits by multiplying with the 64-bit FNV-1 prime.
             // This avoids an inherent affinity between the LSB of a bucket's bits
             // and the stripe an operation ends up on.
-            return bucket.getBucketId().getRawId() * 1099511628211ULL;
+            return bucket.getBucketId().getId() * 1099511628211ULL;
```

Comment:
What is the difference ?

Reply:
Mostly a case of casual paranoia to ensure we always route to the exact same stripe for a given unique bucket. `getId()` is guaranteed to strip away any unused bits from the raw 64-bit ID representation (usually received over the wire) so that you get a canonical representation back. `getRawId()` does not have this property.